book/performance.qmd

```diff
@@ -492,14 +528,8 @@ table(task_grp$data(rows = r$test_set(2), cols = "year"))
 table(task_grp$data(rows = r$test_set(3), cols = "year"))
```

Comment:
Don't need to show all 3, one is enough

Reply:
Reduced to only show the first test set.